docs/operation/upgrade.rst

```diff
@@ -17,17 +19,33 @@ Supported Versions
 
 Upgrade Pre-requisites
 **********************
-Prior to beginning the upgrade steps listed below, make sure to complete the
+Prior to beginning the upgrade step listed below, make sure to complete the
 pre-requisites listed in :doc:`/operation/preparation`.
 
+To try out the feasibility of your upgrade demands, run the following command.
+This will simulate the upgrade procedure and provide an overview of the changes
+to be carried out in your MetalK8s cluster.
+
+   .. code::
+
+     /srv/scality/metalk8s-X.X.X/upgrade.sh --destination-version <destination_version> --dry-run --verbose
+
 Upgrade Steps
 *************
-Ensure that the pre-requisites above have been met before you make
+Ensure that the upgrade pre-requisites above have been met before you make
 any step further.
 
+To upgrade a MetalK8s cluster, run the utility script shipped
+with the **new** version you want to upgrade to providing it with the
+destination version:
 
 - From the :term:`Bootstrap node`, launch the upgrade.
 
+.. important::
+
+    The version prefix metalk8s-**X.X.X** as used below during a MetalK8s
+    upgrade must be the lastest available version you would like to upgrade to.
```

Comment:
must be the latest available (stable?) version you wish to upgrade to.

Reply:
would prefer to stick to ```lastest available or new```  for the following reasons.
To run upgrades
- We somehow add the ISO images of the two versions in the cluster hence the phrase lastest available.
That is if i want to upgrade from 2.3 to 2.4, then these two ISO images need to be present in the cluster.

Reply:
would rather use ```new```